src/clj/game/cards/identities.clj

```diff
@@ -29,7 +29,46 @@
 
 ;;; Card definitions
 (def cards-identities
-  {"Adam: Compulsive Hacker"
+  {
+   "419: Amoral Scammer"
+   {:events {:corp-install
+             {:delayed-completion true
+              :req (req (first-event? state :corp :corp-install))
+              :once :per-turn
+              :effect
+              (req (show-wait-prompt state :corp "Runner to use 419: Amoral Scammer")
+                     (let [itarget target]
+                       (continue-ability
+                         state side
+                         {:optional
+                          {:prompt "Expose installed card unless Corp pays 1 [Credits]?"
+                           :player :runner
+                           :no-ability {:effect (req (clear-wait-prompt state :corp))}
+                           :yes-ability
+                           {:effect (req (clear-wait-prompt state :corp)
+                                         (if (not (can-pay? state :corp nil :credit 1))
+                                           (do
+                                             (toast state :corp "Cannot afford to pay 1 [Credits] to block card exposure" "info")
+                                             (expose state side itarget))
```

Comment:
Since expose is a preventable (awaitable) effect, you should pass `eid` to `expose`, which will also mark the overall effect as completed in this branch.